Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -158,15 +158,25 @@ public final class PropertyVisitor: SyntaxVisitor {
   }
 
   private func findInitializerDescription(from node: PatternBindingListSyntax) -> String? {
-    let initializerClauseSyntaxes = node.compactMap { $0.initializer }
+    let initializerClauseSyntaxes = node.compactMap { $0.initializer?.withEqual(nil) }
     assert(initializerClauseSyntaxes.count <= 1, "A property should have at most one initializer.")
     return initializerClauseSyntaxes.first?.description
   }
 
   private func findCodeBlockDescription(from node: PatternBindingListSyntax) -> String? {
-    let accessors = node.compactMap { $0.accessor }
-    assert(accessors.count <= 1, "A property should have at most one accessor.")
-    return accessors.first?.description
+    guard
```

Comment:
@fdiaz and I spoke and felt that if we are removing the `=` we should also remove the `{` and `}` from computed properties. In the future we could make a sub-visitor to clean up some of this logic.

Reply:
https://github.com/fdiaz/SwiftInspector/pull/112/commits/1bbfbc090e82922701ff2828c8c6f87342ab1de2